kubernetes/.gitignore

```diff
@@ -0,0 +1,20 @@
+# Mac OS X files
```

Comment:
We already have a base `.gitignore`. Please add to that if needed.

Reply:
Sure. I will move to use top level `.gitignore`.